deepmd/dpmodel/model/pairtab_atomic_model.py

```diff
@@ -82,6 +82,9 @@ def fitting_output_def(self) -> FittingOutputDef:
     def get_rcut(self) -> float:
         return self.rcut
 
+    def get_type_map(self) -> Optional[List[str]]:
+        return None
```

Comment:
should have a type map i guess.

Reply:
There is no interface for users to assign a type map for pairtab atomic model...

Reply:
The same suggestion